docs/user/trusted-publishers/using-a-publisher.md

```diff
@@ -209,3 +211,184 @@ below describe the setup process for each supported trusted publisher.
             with:
               password: ${{ steps.mint-token.outputs.api-token }}
     ```
+
+=== "Google Cloud"
+
+    You can use the <https://pypi.org/project/id/> tool to automatically detect
+    and produce OIDC credentials on Google Cloud services.
+
+    First, ensure that `id` and `twine` are installed in the environment you
+    plan to publish from:
+
+    ```
+    $ python -m pip install -U id twine
```

Comment:
strip out leading `$` characters from commands to reduce copy&paste errors

Reply:
d46b615